src/Test.Utilities/CodeMetricsTestsBase.cs

```diff
@@ -13,7 +15,7 @@ namespace Test.Utilities.CodeMetrics
 {
     public abstract class CodeMetricsTestBase
     {
-        private static readonly CompilationOptions s_CSharpDefaultOptions = new Microsoft.CodeAnalysis.CSharp.CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary);
+        private static readonly CompilationOptions s_CSharpDefaultOptions = BuildDefaultCSharpOptions();
```

Comment:
It will also be fine to move `Test.Utilities.csproj` to a newer Microsoft.CodeAnalysis reference.